lunes_cms/static/css/feedback.css

```diff
@@ -8,34 +8,48 @@ table {
   min-width: 1000px;
 }
 
+
+.table-striped tbody tr:nth-of-type(odd){
+  background-color: #ffffff;
+}
+
+thead th .text{
+  color: #001f3f;
+}
+
+thead tr {
+  background-color: #e2e1e1;
+  font-weight: bold;
+}
+
 /* Checkbox column */
 thead th:nth-child(1) {
-  width: 40px;
+  width: 3%;
 }
 
 /* Comment column */
 thead th:nth-child(2) {
-  width: auto;
+  width: 25%;
 }
 
 /* Related object column */
 thead th:nth-child(3) {
-  width: 250px;
+  width: 19%;
```

Comment:
19% seems a bit abitrary. Why did you choose this number?

Reply:
To make it to 100% 😅  3 + 25 + 19 + 15 + 19 + 19 = 100